build.gradle

```diff
@@ -113,4 +113,6 @@ nexusPublishing {
 nexusStaging {
     username = project.hasProperty('sonatypeUsername') ? project.property('sonatypeUsername') : ''
     password = project.hasProperty('sonatypePassword') ? project.property('sonatypePassword') : ''
+    numberOfRetries = 30
+    delayBetweenRetriesInMillis = 3000
```

Comment:
isn't this a bit short? In case there is an issue at sonatype, it typically last for a longer time.